modmail/utils/extensions.py

```diff
@@ -35,7 +35,7 @@ def determine_bot_mode() -> int:
     """
     bot_mode = 0
     for mode in BotModes:
-        if getattr(CONFIG.dev.mode, unqualify(str(mode)).lower(), True):
+        if getattr(config().user.dev.mode, unqualify(str(mode)).lower(), True):
```

Comment:
I don't like the (). why not just call it once?

Reply:
Configuration can change during execution with the configuration manager. This means that it will always grab a new configuration variable before using it.

Reply:
nvm, I understand now.

Reply:
resolved in 3d1986d82bd